README.md

```diff
@@ -1252,6 +1252,7 @@ _Related: [Wikis](#wikis)_
 - [sysPass](https://www.syspass.org/) - Multiuser password management system. ([Demo](https://demo.syspass.org/), [Source Code](https://github.com/nuxsmin/sysPass)) `GPL-3.0` `PHP`
 - [Teampass](https://teampass.net/) - Password manager dedicated for managing passwords in a collaborative way. One symmetric key is used to encrypt all shared/team passwords and stored server side in a file and the database. works on any server Apache, MySQL and PHP. ([Source Code](https://github.com/nilsteampassnet/TeamPass)) `GPL-3.0` `PHP`
 - [Vaultwarden](https://github.com/dani-garcia/vaultwarden) - Lightweight Bitwarden server API implementation written in Rust. `GPL-3.0` `Rust`
+- [vaults](https://github.com/MatrixEternal/vaults) - Minimalistic private password manager featuring client-side encryption, vaults, folders and more. Written in Vue.js 3 & Symfony(PHP). `GPL-3.0` `PHP`
```

Comment:
please use the description from the project's README `Password manager featuring client side AES-256 encryption, PBKDF2 hashing, vaults, password generation & more.` and remove details about frameworks/Vue/Symfony

Reply:
Thank you for the review, I have updated the description.